src/_tkagg.cpp

```diff
@@ -232,6 +264,90 @@ static PyMethodDef functions[] = {
     { NULL, NULL } /* sentinel */
 };
 
+#ifdef DYNAMIC_TKINTER
+// Functions to fill global TCL / Tk function pointers from tkinter module.
+
+#include <dlfcn.h>
+
+#if PY3K
+#define TKINTER_PKG "tkinter"
+#define TKINTER_MOD "_tkinter"
+// From module __file__ attribute to char *string for dlopen.
+#define FNAME2CHAR(s) (PyBytes_AsString(PyUnicode_EncodeFSDefault(s)))
```

Comment:
This isn't exception proof -- `PyUnicode_EncodeFSDefault` could raise an exception here.  Maybe just make a regular old function for this purpose?